core/cmd/local_client_test.go

```diff
@@ -108,6 +113,7 @@ ETH_CHAIN_ID: 0
 DEFAULT_HTTP_LIMIT: 32768
 DEFAULT_HTTP_TIMEOUT: 15s
 CHAINLINK_DEV: true
+SHUTDOWN_GRACE_PERIOD: 30s
```

Comment:
Probably it needs to be slightly less, K8S default is 30s I believe but I have also seen docker documentation that lists 10s.

The application should not take longer than this to shutdown under any reasonable circumstance, so I think setting this to `8s` or so is fine.

Reply:
Note: this is for go tests only. See `NewTestGeneralConfig` that overrides this to 30s only for tests.